app/main/plugins/core/basic-apps/linux.js

```diff
@@ -0,0 +1,71 @@
+import { remote } from 'electron'
+import path from 'path'
+import fs from 'fs'
+import uniq from 'lodash/uniq'
+import { shellCommand } from 'cerebro-tools'
+
+let appDirs = [
+  path.join(remote.app.getPath('home'), '.local', 'share'),
+  path.join('usr', 'share'),
+  path.join('usr', 'share', 'ubuntu'),
+  path.join('usr', 'share', 'gnome'),
+  path.join('usr', 'local', 'share'),
+  path.join('var', 'lib', 'snapd', 'desktop')
```

Comment:
But even then, even if I remove apps_cache.json, the apps from `/usr/share/applications` are missing...

I'll try to get a proper environment and have a closer look.